Update gateway-migration.md (#85513)
Azure/azure-content: articles/azure-sql/database/gateway-migration.md

# Azure SQL Database traffic migration to newer Gateways
[!INCLUDE[appliesto-sqldb](../includes/appliesto-sqldb.md)]

Microsoft periodically refreshes hardware to optimize the customer experience. During these refreshes, Azure adds gateways built on newer hardware generations, migrates traffic to them, and eventually decommissions gateways built on older hardware in some regions.

> To avoid service disruptions during refreshes, allow the communication with SQL Gateway IP subnet ranges for the region. Review [SQL Gateway IP subnet ranges](connectivity-architecture#gateway-ip-addresses) and include the ranges for your region.

Reviewer: ```suggestion
To avoid service disruptions during refreshes, allow the communication with SQL Gateway IP subnet ranges for the region. Review [SQL Gateway IP subnet ranges](connectivity-architecture.md#gateway-ip-addresses) and include the ranges for your region.
```

> To avoid service disruptions during refreshes, allow the communication with SQL Gateway IP subnet ranges for the region. Review [SQL Gateway IP subnet ranges](https://docs.microsoft.com/en-us/azure/azure-sql/database/connectivity-architecture#gateway-ip-addresses) and include the ranges for your region.

Reviewer: ```suggestion
To avoid service disruptions during refreshes, allow the communication with SQL Gateway IP subnet ranges for the region. Review [SQL Gateway IP subnet ranges](connectivity-architecture#gateway-ip-addresses) and include the ranges for your region.
```

Customers can [use the Azure portal to set up activity log alerts](../../service-health/alerts-activity-log-service-notifications-portal.md).

> The most up-to-date information will be maintained in the [Azure SQL Database gateway IP addresses](connectivity-architecture.md#gateway-ip-addresses) table.

Reviewer: ```suggestion
```

# Azure SQL Database traffic migration to newer Gateways
[!INCLUDE[appliesto-sqldb](../includes/appliesto-sqldb.md)]

> As Azure infrastructure improves, Microsoft will periodically refresh hardware to ensure we provide the best possible customer experience. In the coming months, we plan to add gateways built on newer hardware generations, migrate traffic to them, and eventually decommission gateways built on older hardware in some regions. 

Reviewer: ```suggestion
Microsoft periodically refreshes hardware to optimize the customer experience. During these refreshes, Azure adds gateways built on newer hardware generations, migrates traffic to them, and eventually decommissions gateways built on older hardware in some regions.
```

> As Azure infrastructure improves, Microsoft will periodically refresh hardware to ensure we provide the best possible customer experience. In the coming months, we plan to add gateways built on newer hardware generations, migrate traffic to them, and eventually decommission gateways built on older hardware in some regions. 

Reviewer: ```suggestion
Microsoft periodically refreshes hardware to optimize the customer experience. During these refreshes, Azure periodically adds gateways built on newer hardware generations, migrates traffic to them, and eventually decommissions gateways built on older hardware in some regions. 
```

> We have published [SQL Gateway IP subnet ranges](https://docs.microsoft.com/en-us/azure/azure-sql/database/connectivity-architecture#gateway-ip-addresses) and new SQL Gateway IP addresses will be picked up from this range with no notification sent in advance. Customer need to allow the communication with SQL Gateway IP subnet ranges for the region to avoid service disruptions. 

Reviewer: ```suggestion
To avoid service disruptions during refreshes, allow the communication with SQL Gateway IP subnet ranges for the region. Review [SQL Gateway IP subnet ranges](connectivity-architecture#gateway-ip-addresses) and include the ranges for your region.
```

> We have published [SQL Gateway IP subnet ranges](https://docs.microsoft.com/en-us/azure/azure-sql/database/connectivity-architecture#gateway-ip-addresses) and new SQL Gateway IP addresses will be picked up from this range with no notification sent in advance. Customer need to allow the communication with SQL Gateway IP subnet ranges for the region to avoid service disruptions. Customers can [use the Azure portal to set up activity log alerts](../../service-health/alerts-activity-log-service-notifications-portal.md).

Reviewer: ```suggestion

To avoid service disruptions during refreshes, allow the communication with SQL Gateway IP subnet ranges for the region. Review [SQL Gateway IP subnet ranges](https://docs.microsoft.com/en-us/azure/azure-sql/database/connectivity-architecture#gateway-ip-addresses) and include the ranges for your region.

Customers can [use the Azure portal to set up activity log alerts](../../service-health/alerts-activity-log-service-notifications-portal.md).
```
